### PR TITLE
[FR] add `history_of_osu/osu!wiki`

### PR DESCRIPTION
**Pull request by Biokax**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/History_of_osu!/osu!_wiki/fr.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+| Date | Description |
+| :-- | :-- |
+| 26-08-2016 | [peppy a créé le repository osu-wiki](https://github.com/ppy/osu-wiki/tree/3433cbeeda9303a470647cad1c338d43f4272a2e). craftu et Xyloo ont porté le contenu du MediaWiki dans le repository en une semaine. |
+| 09-2016 | La plupart des pages, excepté les images ont été déplacés dans le repository, avec des erreurs de syntaxe et de traduction non résolues découlant des différences linguistiques entre Markdown et MediaWiki. |
+| 01-2017 | osu!web supporte le wiki osu!, fait par nanaya. |
+| 05-01-2017 | [Le projet osu! wiki suggéré par Ephemeralis](https://github.com/ppy/osu-wiki/projects/1). |
+| 26-01-2017 | Le wiki osu! a été mentionné dans le blog de ppy. |
+| 22-05-2017 | [Les archives osu!news sont maintenant stockées sur le wiki osu! (sous format texte uniquement)](https://github.com/ppy/osu-wiki/pull/252). |
+| 30-05-2017 | [La redirection de page est finalement implémentée dans le wiki osu!](https://github.com/ppy/osu-web/pull/1144) |
+| 06-2017 | TPGPL est promu avec des accès en écriture sur le repository osu-wiki. La fonction de recherche pour le wiki osu! est implémentée.
```
Comment (Biokax) [inline wiki/History_of_osu!/osu!_wiki/fr.md:35]:
```suggestion
| 06-2017 | TPGPL est promu avec des accès en écriture sur le repository osu-wiki. La fonction de recherche pour le wiki osu! est implémentée. |
```

**Review by Biokax — COMMENTED** [review]
attempt to fix CI failure

File: wiki/History_of_osu!/osu!_wiki/fr.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+outdated: true
+---
+
+# L'histoire du osu! wiki
```
Comment (Ganondorf-1) [inline wiki/History_of_osu!/osu!_wiki/fr.md:5]:
```suggestion
# L'histoire de l'osu! wiki
```

File: wiki/History_of_osu!/osu!_wiki/fr.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+## MediaWiki (2011 - 2017)
+
+![](img/ow_MW.jpg "Screenshot de la page complète du osu! wiki en MediaWiki, prit le 17-02-2017")
```
Comment (Ganondorf-1) [inline wiki/History_of_osu!/osu!_wiki/fr.md:9]:
```suggestion
![](img/ow_MW.jpg "Capture d'écran de la page complète de l'osu! wiki en MediaWiki, prise le 17 février 2017")
```

File: wiki/History_of_osu!/osu!_wiki/fr.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+| Date, Temps | Description |
```
Comment (Ganondorf-1) [inline wiki/History_of_osu!/osu!_wiki/fr.md:11]:
```suggestion
| Date, Heure | Description |
```

File: wiki/History_of_osu!/osu!_wiki/fr.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+| :-- | :-- |
+| 05/12/2011 T08:13Z | Première modification d'osu! wiki par Ephemeral. |
+| 06/12/2011 | [osu! wiki est maintenant en ligne !](https://osu.ppy.sh/community/forums/topics/68525). |
+| 11/2012 | osu! wiki a maintenant 3 administrateurs (Dellirium, Repflez). Précédemment, peppy était le seul admin. |
+| 27/01/2013 T06:12Z | peppy a intégré le wiki osu! dans le design du site principal. [Lien vers le post de peppy](https://osu.ppy.sh/community/forums/topics/68479/start=124). |
```
Comment (Ganondorf-1) [inline wiki/History_of_osu!/osu!_wiki/fr.md:16]:
```suggestion
| 27/01/2013 T06:12Z | peppy a intégré l'osu! wiki dans le design du site principal. [Lien vers le post de peppy](https://osu.ppy.sh/community/forums/topics/68479/start=124). |
```

File: wiki/History_of_osu!/osu!_wiki/fr.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+| 2014 | Le wiki osu! est devenu la source d'information principale d'osu!, remplaçant la plupart des copies web, tel que [L'équipe](/wiki/People/The_Team) ainsi que les [Règles](/wiki/Rules). Également, le wiki osu! possède maintenant 5 administrateurs (Loctav, deadbeat). |
```
Comment (Ganondorf-1) [inline wiki/History_of_osu!/osu!_wiki/fr.md:17]:
```suggestion
| 2014 | L'osu! wiki est devenu la source d'information principale d'osu!, remplaçant la plupart des copies web, telles que [L'équipe](/wiki/People/The_Team) ainsi que les [Règles](/wiki/Rules). Par ailleurs, l'osu! wiki possède maintenant 5 administrateurs (Loctav, deadbeat). |
```

File: wiki/History_of_osu!/osu!_wiki/fr.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+| 2015 | L'osu! wiki a reçus une traduction majeur des autres langues. Dellirium a été remplacé par Shiro en tant qu'un des administrateurs du osu! wiki. |
```
Comment (Ganondorf-1) [inline wiki/History_of_osu!/osu!_wiki/fr.md:18]:
```suggestion
| 2015 | L'osu! wiki a reçu une traduction majeure des autres langues. Dellirium a été remplacé par Shiro en tant qu'un des administrateurs du osu! wiki. |
```

File: wiki/History_of_osu!/osu!_wiki/fr.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+| 22/02/2016 | Le wiki osu! a maintenant 8 administrateurs (Ukami and Galkan). |
```
Comment (Ganondorf-1) [inline wiki/History_of_osu!/osu!_wiki/fr.md:19]:
```suggestion
| 22/02/2016 | L'osu! wiki a maintenant 8 administrateurs (Ukami et Galkan). |
```

File: wiki/History_of_osu!/osu!_wiki/fr.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+| 01/04/2016 | Le wiki osu! a maintenant 9 administrateurs (Nathanael). |
```
Comment (Ganondorf-1) [inline wiki/History_of_osu!/osu!_wiki/fr.md:20]:
```suggestion
| 01/04/2016 | L'osu! wiki a maintenant 9 administrateurs (Nathanael). |
```

File: wiki/History_of_osu!/osu!_wiki/fr.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+| 30/08/2016 | Le wiki osu! fonctionnant sur MediaWiki a été déprécié en faveur de la version fonctionnant sur le repository Github. Le wiki actuel d'osu! va rester jusqu'à ce que la version Github soit prête à être déployée avec toutes les pages, ainsi que les images portées. |
```
Comment (Ganondorf-1) [inline wiki/History_of_osu!/osu!_wiki/fr.md:21]:
```suggestion
| 30/08/2016 | L'osu! wiki fonctionnant sur MediaWiki a été déprécié en faveur de la version fonctionnant sur le repository GitHub. Le wiki actuel d'osu! va rester jusqu'à ce que la version GitHub soit prête à être déployée avec toutes les pages, ainsi que les images portées. |
```

File: wiki/History_of_osu!/osu!_wiki/fr.md [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+| 06/2017 | Le wiki osu! fonctionnant sur MediaWiki est officiellement déprécié. Le lien pointe maintenant vers la nouvelle interface osu-web utilisant les ressources du wiki osu! faites à partir du repository GitHub. [Une backup temporaire du wiki osu! sans les fonctionnalités de MediaWiki peut-être trouvée ici.](https://osu.ppy.sh/old-wiki/Main_Page) |
```
Comment (Ganondorf-1) [inline wiki/History_of_osu!/osu!_wiki/fr.md:22]:
```suggestion
| 06/2017 | L'osu! wiki fonctionnant sur MediaWiki est officiellement déprécié. Le lien pointe maintenant vers la nouvelle interface osu-web utilisant les ressources de l'osu! wiki faites à partir du repository GitHub. [Une backup temporaire de l'osu! wiki sans les fonctionnalités de MediaWiki peut-être trouvée ici.](https://osu.ppy.sh/old-wiki/Main_Page) |
```

File: wiki/History_of_osu!/osu!_wiki/fr.md [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+
+## repository GitHub (2016 - maintenant)
```
Comment (Ganondorf-1) [inline wiki/History_of_osu!/osu!_wiki/fr.md:24]:
```suggestion
## Repository GitHub (2016 - maintenant)
```

File: wiki/History_of_osu!/osu!_wiki/fr.md [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+
+| Date | Description |
+| :-- | :-- |
+| 26/08/2016 | [peppy a créé le repository osu-wiki](https://github.com/ppy/osu-wiki/tree/3433cbeeda9303a470647cad1c338d43f4272a2e). craftu et Xyloo ont porté le contenu du MediaWiki dans le repository en une semaine. |
+| 09/2016 | La plupart des pages, excepté les images ont été déplacés dans le repository, avec des erreurs de syntaxe et de traduction non résolues découlant des différences linguistiques entre Markdown et MediaWiki. |
```
Comment (Ganondorf-1) [inline wiki/History_of_osu!/osu!_wiki/fr.md:29]:
```suggestion
| 09/2016 | La plupart des pages, excepté les images, ont été déplacées dans le repository, avec des erreurs de syntaxe et de traduction non résolues découlant des différences linguistiques entre Markdown et MediaWiki. |
```

File: wiki/History_of_osu!/osu!_wiki/fr.md [hunk 14]
```diff
@@ -0,0 +1,36 @@
...
+| 01/2017 | osu!web supporte le wiki osu!, fait par nanaya. |
```
Comment (Ganondorf-1) [inline wiki/History_of_osu!/osu!_wiki/fr.md:30]:
```suggestion
| 01/2017 | Grâce à nanaya, osu!web prend en charge l'osu! wiki. |
```

File: wiki/History_of_osu!/osu!_wiki/fr.md [hunk 15]
```diff
@@ -0,0 +1,36 @@
...
+| 05/01/2017 | [Le projet osu! wiki suggéré par Ephemeralis](https://github.com/ppy/osu-wiki/projects/1). |
+| 26/01/2017 | Le wiki osu! a été mentionné dans le blog de ppy. |
```
Comment (Ganondorf-1) [inline wiki/History_of_osu!/osu!_wiki/fr.md:32]:
```suggestion
| 26/01/2017 | L'osu! wiki a été mentionné dans le blog de ppy. |
```

File: wiki/History_of_osu!/osu!_wiki/fr.md [hunk 16]
```diff
@@ -0,0 +1,36 @@
...
+| 22/05/2017 | [Les archives osu!news sont maintenant stockées sur le wiki osu! (sous format texte uniquement)](https://github.com/ppy/osu-wiki/pull/252). |
```
Comment (Ganondorf-1) [inline wiki/History_of_osu!/osu!_wiki/fr.md:33]:
```suggestion
| 22/05/2017 | [Les archives osu!news sont maintenant stockées sur l'osu! wiki (sous format texte uniquement)](https://github.com/ppy/osu-wiki/pull/252). |
```

File: wiki/History_of_osu!/osu!_wiki/fr.md [hunk 17]
```diff
@@ -0,0 +1,36 @@
...
+| 30/05/2017 | [La redirection de page est finalement implémentée dans le wiki osu!](https://github.com/ppy/osu-web/pull/1144) |
```
Comment (Ganondorf-1) [inline wiki/History_of_osu!/osu!_wiki/fr.md:34]:
```suggestion
| 30/05/2017 | [La redirection de page est finalement implémentée dans l'osu! wiki](https://github.com/ppy/osu-web/pull/1144) |
```

File: wiki/History_of_osu!/osu!_wiki/fr.md [hunk 18]
```diff
@@ -0,0 +1,36 @@
...
+| 06/2017 | TPGPL est promu avec des accès en écriture sur le repository osu-wiki. La fonction de recherche pour le wiki osu! est implémentée. |
```
Comment (Ganondorf-1) [inline wiki/History_of_osu!/osu!_wiki/fr.md:35]:
```suggestion
| 06/2017 | TPGPL est promu avec des accès en écriture sur le repository osu-wiki. La fonction de recherche pour l'osu! wiki est implémentée. |
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

File: wiki/History_of_osu!/osu!_wiki/fr.md [hunk 19]
```diff
@@ -0,0 +1,36 @@
...
+![](img/ow_MW.jpg "Capture d'écran de la page complète de l'osu! wiki en MediaWiki, prise le 17 février 2017")
+
+| Date, Heure | Description |
+| :-- | :-- |
+| 05/12/2011 T08:13Z | Première modification d'osu! wiki par Ephemeral. |
+| 06/12/2011 | [osu! wiki est maintenant en ligne !](https://osu.ppy.sh/community/forums/topics/68525). |
+| 11/2012 | osu! wiki a maintenant 3 administrateurs (Dellirium, Repflez). Précédemment, peppy était le seul admin. |
+| 27/01/2013 T06:12Z | peppy a intégré l'osu! wiki dans le design du site principal. [Lien vers le post de peppy](https://osu.ppy.sh/community/forums/topics/68479/start=124). |
+| 2014 | L'osu! wiki est devenu la source d'information principale d'osu!, remplaçant la plupart des copies web, telles que [L'équipe](/wiki/People/The_Team) ainsi que les [Règles](/wiki/Rules). Par ailleurs, l'osu! wiki possède maintenant 5 administrateurs (Loctav, deadbeat). |
+| 2015 | L'osu! wiki a reçu une traduction majeure des autres langues. Dellirium a été remplacé par Shiro en tant qu'un des administrateurs du osu! wiki. |
```
Comment (Ganondorf-1) [inline wiki/History_of_osu!/osu!_wiki/fr.md:18]:
```suggestion
| 2015 | L'osu! wiki a reçu une traduction majeure des autres langues. Dellirium a été remplacé par Shiro en tant qu'un des administrateurs de l'osu! wiki. |
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

**Review by Ganondorf-1 — DISMISSED** [review]

File: wiki/History_of_osu!/osu!_wiki/fr.md [hunk 20]
```diff
@@ -0,0 +1,36 @@
...
+
+| Date | Description |
+| :-- | :-- |
+| 26/08/2016 | [peppy a créé le repository osu-wiki](https://github.com/ppy/osu-wiki/tree/3433cbeeda9303a470647cad1c338d43f4272a2e). craftu et Xyloo ont porté le contenu du MediaWiki dans le repository en une semaine. |
+| 09/2016 | La plupart des pages, excepté les images, ont été déplacées dans le repository, avec des erreurs de syntaxe et de traduction non résolues découlant des différences linguistiques entre Markdown et MediaWiki. |
+| 01/2017 | Grâce à nanaya, osu!web prend en charge l'osu! wiki. |
+| 05/01/2017 | [Le projet osu! wiki suggéré par Ephemeralis](https://github.com/ppy/osu-wiki/projects/1). |
+| 26/01/2017 | L'osu! wiki a été mentionné dans le blog de ppy. |
+| 22/05/2017 | [Les archives osu!news sont maintenant stockées sur l'osu! wiki (sous format texte uniquement)](https://github.com/ppy/osu-wiki/pull/252). |
+| 30/05/2017 | [La redirection de page est finalement implémentée dans l'osu! wiki](https://github.com/ppy/osu-web/pull/1144) |
```
Comment (Walavouchey) [inline wiki/History_of_osu!/osu!_wiki/fr.md:34]:
```suggestion
| 30/05/2017 | [La redirection de page est finalement implémentée dans l'osu! wiki.](https://github.com/ppy/osu-web/pull/1144) |
```

File: wiki/History_of_osu!/osu!_wiki/fr.md [hunk 21]
```diff
@@ -0,0 +1,36 @@
...
+# L'histoire de l'osu! wiki
+
+## MediaWiki (2011 - 2017)
+
+![](img/ow_MW.jpg "Capture d'écran de la page complète de l'osu! wiki en MediaWiki, prise le 17 février 2017")
+
+| Date, Heure | Description |
+| :-- | :-- |
+| 05/12/2011 T08:13Z | Première modification d'osu! wiki par Ephemeral. |
+| 06/12/2011 | [osu! wiki est maintenant en ligne !](https://osu.ppy.sh/community/forums/topics/68525). |
```
Comment (Walavouchey) [inline wiki/History_of_osu!/osu!_wiki/fr.md:14]:
```suggestion
| 06/12/2011 | [osu! wiki est maintenant en ligne !](https://osu.ppy.sh/community/forums/topics/68525) |
```

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**